stustapay/core/service/customer/customer.py

```diff
@@ -52,38 +54,45 @@ class CustomerBank(BaseModel):
     donation: float = 0.0
 
 
-async def get_number_of_customers(conn: asyncpg.Connection) -> int:
-    # number of customers with iban not null and balance > 0
-    return await conn.fetchval(
-        "select count(*) from customer c "
-        "where c.iban is not null "
-        "and round(c.balance, 2) > 0 "
-        "and round(c.balance - c.donation, 2) > 0"
+async def get_number_of_payouts(conn: asyncpg.Connection, payout_run_id: Optional[int]) -> int:
+    if payout_run_id is None:
+        return await conn.fetchval("select count(*) from payout where payout_run_id is null")
+    return await conn.fetchval("select count(*) from payout where payout_run_id = $1", payout_run_id)
+
+
+async def create_payout_run(conn: asyncpg.Connection, created_by: str) -> tuple[int, int]:
+    payout_run_id = await conn.fetchval(
+        "insert into payout_run (created_at, created_by) values (now(), $1) returning id", created_by
     )
+    # set payout_run_id for all pending payouts
+    await conn.execute(
+        "update customer_info c set payout_run_id = $1 where c.customer_account_id in "
+        "(select p.customer_account_id from payout p where p.payout_run_id is null)",
+        payout_run_id,
+    )
+    number_of_payouts = await conn.fetchval("select count(*) from payout where payout_run_id = $1", payout_run_id)
+    return payout_run_id, number_of_payouts
 
 
 async def get_customer_bank_data(
-    conn: asyncpg.Connection, max_export_items_per_batch: int, ith_batch: int = 0
-) -> List[CustomerBankData]:
+    conn: asyncpg.Connection, payout_run_id: int, max_export_items_per_batch: int, ith_batch: int = 0
+) -> list[Payout]:
     rows = await conn.fetch(
-        "select c.iban, c.account_name, c.email, c.user_tag_uid, (c.balance - c.donation) as balance "
-        "from customer c "
-        "where c.iban is not null and round(c.balance, 2) > 0 and round(c.balance - c.donation, 2) > 0 "
-        "limit $1 offset $2",
+        "select * from payout where payout_run_id = $1 order by user_tag_uid asc limit $2 offset $3",
```

Comment:
items with `payout_export=false` are not selected here, but where is `payout_export` set to true?